### PR TITLE
✨ Add NewPlant page

### LINKED ISSUE #6: "NewPlant" page
#### Description

The "NewPlant" page gives the user the opportunity to create a new plant for his pocket-garden.
The following options are available for the user:

- Upload an image
- Enter a plant name
- Select the sowing month
- Select a place sunny, half shady or shady
- Select perennial yes or no
- Write a note

#### Screenshots

image

#### Tasks

`Components`

- [ ] Image Upload: ???
- [ ] Plant name: `<Input></input>` 
- [ ] Sowing: `<button>{month}</button>`
- [ ] Place: `<button>{place}</button>`
- [ ] Perennial: `<button></button>`
- [ ] Note: `<Input></input>`

### PR DESCRIPTION
**Pull request by MaxMuecket**
fixes #6

- Add NewPlantPage.tsx
- Add NewPlantPage.stories.tsx
- Add NewPlantPage.module.css
   
- Update DateButton, PlaceButton and PageInput `component`
   
The User can add a new plant to his pocket-garden at the NewPlant page.

https://pocket-garde-new-plant--glrxqo.herokuapp.com/new-plant

### CODE REVIEW

**Review by Niklas-Schaefers — APPROVED** [review]
lgtm 🚀

File: src/app/pages/NewPlantPage/NewPlantPage.module.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
+.container {
+  background-image: url('../../assets/leaf.png');
+  background-repeat: no-repeat;
+  height: 100vh;
+}
+
+.header__title {
+  font-size: 3.438rem;
```
Comment (jujuzitroni) [inline src/app/pages/NewPlantPage/NewPlantPage.module.css:8]: seems very specific, maybe is 3.4rem good enough?

File: src/app/pages/NewPlantPage/NewPlantPage.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  const [name, setName] = useState('');
+  const [firstMonth, setFirstMonth] = useState(0);
+  const [lastMonth, setLastMonth] = useState(0);
+  const [place, setPlace] = useState(0);
+  const [active, setActive] = useState(false);
+  const [note, setNote] = useState('');
+
+  function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+    console.log(name, active, note, firstMonth, lastMonth, place);
```
Comment (jujuzitroni) [inline src/app/pages/NewPlantPage/NewPlantPage.tsx:21]:
```suggestion
   
```

**Review by jujuzitroni — APPROVED** [review]
nice! 🥇

**Pull request merged**